### DISCUSSION TITLE
Is there any way to set year dropdown option size?

### DISCUSSION THREAD

**Discussion by irenefelita192** [easepick/easepick#132] [Q&A]
Hi, I'm using the AmpPlugin with month and year dropdown. My question is, especially for year dropdown, is there any way we can set the size / number of options shown? For example only show 10 list of years for each scroll

<img width="124" alt="Screen Shot 2022-09-16 at 15 08 37" src="https://user-images.githubusercontent.com/42824601/190590839-fb70a5ac-e8b6-4296-8381-10e62990c66d.png">

**Comment by beyer96** [comment]
I guess normally this would be the answer: https://stackoverflow.com/a/23534658

I just tried to implement it, but since whole easepick component is inside shadowRoot, you can't set `size` attribute for this select as you want.. :/

**Reply by wakirin** [reply]
Maybe this will help, but not sure if this is the expected result.
You can find `select` elements in `view` event. https://jsfiddle.net/y1q9cknt/
